all-about-that-bash.sh

```diff
@@ -879,4 +879,25 @@ try-land() {
     git fetch --no-tags origin develop && git rebase origin/develop && arc land --onto develop
 }
 
-trap 'echo -e "${clPURPLE}-- Started at $(date +"%H:%M:%S") --${cLIGHTGRAY}"' DEBUG
\ No newline at end of file
+#pre-requisite: checked out on develop branch
```

Comment:
you can check it by using `$(getCurrentGitBranch)` function, it's provided on the same file.

Reply:
done @haruntvlk